app/lib/feed_manager.rb

```diff
@@ -30,6 +30,7 @@ def push_to_home(account, status)
     return false unless add_to_feed(:home, account.id, status)
     trim(:home, account.id)
     PushUpdateWorker.perform_async(account.id, status.id, "timeline:#{account.id}") if push_update_required?("timeline:#{account.id}")
+    NotifyService.new.call(account, status, :post) unless status.reblog? && status.reblog.account == account
```

Comment:
Wait, so you just duplicate the entire home feed in the notifications? I thought the idea was to opt-in to get notifications like that from *particular users*, like those who post rarely.

This won't work, because it would explode storage requirements (notifications table is a lot spacier than redis feeds). Also, in any case this feels like something that should happen in FanOutOnWriteService rather here.

Reply:
Okay, I confused #6437 and #4890. But even in this case, it's too much work to save home feed items as notifications, unless they need to be shown in the notifications column, which I don't think they do. Currently the code is written so you can send a push notification/e-mail from a Notification object but I think it would be easier to just add special cases for that. After all, a one-off e-mail/push notification does not require a corresponding notification record.

Reply:
I'll rework this patch into a new PR based on you suggestions as my goal is just to send one-off notifications about all new posts without keeping a history. This is the wrong approach for sure.

Anyway, this work became a good base for #4890, maybe I'll pick that too.

Thanks for the review, I'll be back. :)